### DISCUSSION TITLE
How can i add role for superAdmin who will have All Subjects and Actions?

### DISCUSSION THREAD

**Discussion by hrynevychroman** [stalniy/casl#809] [Q&A] [answered]
I Have many Subjects inside my app, and Actions are CRUD standart. So, i need to have super admin, but don't think that return all 100+ permissions are great approach 🙂

**Comment by stalniy** [comment, marked as answer]
`can(“manage”, “all”)`

**Reply by aprilbigby** [reply]
how does this help?

**Reply by stalniy** [reply]
This is definition in AbilityBuilder. action “manage” represents any action, and similarly “all” represents any subject type

**Reply by sachingaikwad123** [reply]
Are these keywords: "manage" and "all" reserved? I did not see any documentation for this. Can I use them safely for "production" environment as well for "super admin" use-case?

Also, I read "Invalid usage" regarding "manage" here: https://casl.js.org/v6/en/guide/define-aliases#invalid-usage . This is confusing to me. Is it saying that in "production env", "manage" will be removed automatically or is this regarding only "manually define action aliases being" removed? Can you please clarify on this?

**Reply by stalniy** [reply]
Yes, they are reserved defaults. But you can provide your own in AbilityOptions, 2nd parameter to ability factories
